src/components/cc-domain-management/cc-domain-management.types.d.ts

```diff
@@ -0,0 +1,83 @@
+export type DomainManagementFormState = DomainManagementFormStateIdle | DomainManagementFormStateAdding;
+
+interface DomainManagementFormStateIdle {
+  type: 'idle';
+  domain: {
+    value: string;
+    error?: FormError | null;
+  };
+  pathPrefix: {
+    value: string;
+  };
+}
+
+interface DomainManagementFormStateAdding {
+  type: 'adding';
+  domain: {
+    value: string;
+    error?: null;
```

Comment:
sometimes you reset errors by setting them to `null`.
Also if you remove the error property from the type here and you try to do something like this : 
`this._getErrorMessage(domain.error)`

TypeScript will throw an error because you are trying to access `error` which does not exist in waiting state. Setting this to `null` fixes that :man_shrugging: